### DISCUSSION TITLE
Specify multiple host connections for PostgreSQL

### DISCUSSION THREAD

**Discussion by Mohamedelh** [prisma/prisma#13485] [Q&A] [answered]
According to [PostgreSQL documentation](https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-MULTIPLE-HOSTS), it is possible in PostgreSQL to specify multiple hosts to connect to. I would like to use this option with Prisma as a part of a failover strategy (in conjunction with the `target_session_attrs` parameter) . I have looked through Prisma documentation and open/closed issues and could only find information about this with MongoDB ([Example](https://github.com/prisma/prisma/issues/11873)). As a result, I am still inconclusive as to whether this is possible or not with Prisma. I tested this by simply inserting a second connection in the connection url, but then I receive the error that the database string is invalid.

Is this already possible in Prisma? If yes, how?

**Comment by nurul3101** [comment, marked as answer]
I don't think this is possible at the moment, I'll still check.
I can't see a Feature Request for it either, do you mind creating it [here](https://github.com/prisma/prisma/issues/new?assignees=&labels=&template=feature_request.md&title=)

**Reply by Mohamedelh** [reply]
Hi @nurul3101, thank you for your response. I have requested a feature for this [here](https://github.com/prisma/prisma/issues/13493)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
